std-bits/table/src/main/java/org/enso/table/expressions/ExpressionVisitorImpl.java

```diff
@@ -67,26 +142,22 @@ public static Value evaluate(
     var context = Context.getCurrent().getBindings("enso");
     final Value module = context.invokeMember("get_module", moduleName);
     final Value type = module.invokeMember("get_type", typeName);
-    Function<String, Value> getMethod = name -> module.invokeMember("get_method", type, name);
+    final var setVariableArgumentFunctions =
+        new HashSet<>(Arrays.asList(variableArgumentFunctions));
+    Function<String, Method> getMethod =
```

Comment:
feels like we might want to memoization fetching the methods?

Reply:
We should measure to see if the added complexity is worth it. I'll create a rough timing test next week and see how long this actually takes. My assumption was it was all relatively cheap, but we should measure.